sources/osgViewer/Viewer.js

```diff
@@ -404,18 +408,12 @@ Viewer.prototype = MACROUTILS.objectInherit( View.prototype, {
         var self = this;
         var render = function () {
             if ( !self.done() ) {
-
-                var isPresentingVR = self._hmd && self._hmd.isPresenting;
-
-                if ( isPresentingVR )
+                if ( self._hmd && self._hmd.isPresenting )
```

Comment:
> The signatures are a bit different (the vr requestAnimation doesn't take a html 2nd element in argument for example).

In fact that second element is optional, and only for chrome webkit AFAIK( https://developer.mozilla.org/es/docs/Web/API/Window/requestAnimationFrame ). 

> Also, if I'm not mistaken, isPresenting must be read at each frame as it's the only way to know if we have to render in vr or not.

I don't know, it that is really the case then it does not make sense to avoid the conditional. But from what I've read : " When the VRDisplay is not presenting a scene, this is functionally equivalent to Window.requestAnimationFrame."
(https://developer.mozilla.org/en-US/docs/Web/API/VRDisplay/requestAnimationFrame)

> What's the issue with the conditional? It takes 0 perf and should be completely transparent in non vr.

It's just that I don't feel comfortable doing an extra conditional every frame if it is really not needed. It does not look like a best practice from the design point of view.

I could be wrong of course, but in my opinion we can avoid that conditional. 

Thoughts?